nxdrive/engine/dao/sqlite.py

```diff
@@ -730,6 +730,28 @@ def _migrate_db(self, cursor: Cursor, version: int) -> None:
                 )
             self.store_int(SCHEMA_VERSION, 8)
 
+        if version < 9:
+            # Change Downloads.path and Uploads.path database field types.
+            # SQLite does not support column type update, we need to recreate
+            # a new one and insert back old data.
+
+            # Make a copy of the Upload and Download table
+            cursor.execute("ALTER TABLE Uploads RENAME TO Uploads_backup;")
+            cursor.execute("ALTER TABLE Downloads RENAME TO Downloads_backup;")
+
+            # Create again the tables, with up-to-date columns
+            self._create_transfer_tables_v2(cursor)
```

Comment:
Just fix the types in `_create_transfer_tables()` ;)

Reply:
Done